tests/ha/single_pod_restart/test_server_pod_restart.py

```diff
@@ -717,3 +718,472 @@ def test_reads_during_server_pod_restart(self):
+            del_resp = del_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        if not del_resp:
+            assert_utils.assert_true(False, "Background process failed to do deletes")
+        fail_del_bkt = del_resp[1]
+        rem_bkts_aftr_del = s3_test_obj.bucket_list()[1]
+        assert_utils.assert_false(len(fail_del_bkt),
+                                  "Bucket deletion failed when server pod restart"
+                                  f" {fail_del_bkt}")
+        assert_utils.assert_equals(len(rem_bkts_aftr_del), del_bucket, "All buckets are expected "
+                                                                       "to be deleted while server "
+                                                                       "pod restarted")
+        LOGGER.info("Step 6.1: Verified status for In-flight DELETEs")
+        LOGGER.info("Step 6.2: Verify status for In-flight WRITEs")
+        responses_wr = dict()
+        while len(responses_wr) != 2:
+            responses_wr = output_wr.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        pass_logs = list(x[1] for x in responses_wr["pass_res"])
+        fail_logs = list(x[1] for x in responses_wr["fail_res"])
+        resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
+        assert_utils.assert_false(len(resp[1]), "WRITEs before and after pod deletion are "
+                                                "expected to pass.Logs which contain failures:"
+                                                f"{resp[1]}")
+        resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs)
+        assert_utils.assert_false(len(resp[1]), "In-flight WRITEs Logs which contain failures: "
+                                                f"{resp[1]}")
+        LOGGER.info("Step 6.2: Verified status for In-flight WRITEs")
+        LOGGER.info("Step 6.3: Verify status for In-flight READs/Verify DI")
+        responses_rd = dict()
+        while len(responses_rd) != 2:
+            responses_rd = output_rd.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        pass_logs = list(x[1] for x in responses_rd["pass_res"])
+        fail_logs = list(x[1] for x in responses_rd["fail_res"])
+        resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
+        assert_utils.assert_false(len(resp[1]), "READs/VerifyDI logs which contain failures:"
+                                                f"{resp[1]}")
+        resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs)
+        assert_utils.assert_false(len(resp[1]), "READs/VerifyDI Logs which contain failures: "
+                                                f"{resp[1]}")
+        LOGGER.info("Step 6.3: Verified status for In-flight READs/Verify DI")
+        LOGGER.info("Step 6: Verified status for In-flight READs/WRITEs/DELETEs while server pod "
+                    "%s was restarted.", pod_name)
+        LOGGER.info("Step 7: Verify READ/Verify for data written in healthy cluster and delete "
```

Comment:
and DELETE

Reply:
delete buckets is written so shud be ok @ApurwaMohite